scriptler/addShellPreBuildStepToProject.groovy

```diff
@@ -0,0 +1,42 @@
+/*** BEGIN META {
+ "name" : "Add a pre-build shell script step",
+ "comment" : "Add a pre-build shell script step to all Maven and Frestyle Projects",
+ "parameters" : ["filePath"],
+ "core": "1.609",
+ "authors" : [
+ { name : "Kuisathaverat" }
+ ]
+ } END META**/
+
+import hudson.util.*
+import hudson.tasks.*
+import hudson.maven.*
+
+//def filePath = '/path/to/script.sh'
+def scriptFile = new File(filePath)
+
+if(!scriptFile.exists() && scriptFile.isDirectory()) { 
+  throw IOException("Check file name")
```

Comment:
Syntax issue

`throw IOException("Check file name")` should be `throw new IOException("Check file name")`